lib/ansible/modules/network/nxos/nxos_igmp_snooping.py

```diff
@@ -260,6 +273,7 @@ def main():
         if delta:
             command = config_igmp_snooping(delta, existing)
             if command:
+                igmp_snooping_gt_dependency(command, existing, module)
```

Comment:
If I understand correctly, this method is only required for group_timeout right? If yes we can call the method just when group_timeout value is given. `if command and group_timeout`:

Reply:
That's correct. Thanks for the suggestion, I added the check for `group_timeout`.